DevDashboard/cloud/landing/obsidian-terminal/index.html

```
@@ -0,0 +1,821 @@
+          <a href="#pricing"  class="rounded-full px-3.5 py-1.5 transition-colors duration-500 ease-silk hover:bg-white/[0.06] hover:text-zinc-100">Pricing</a>
+        </div>
+
+        <a href="#pricing" class="hidden shrink-0 items-center gap-2 rounded-full bg-zinc-100 py-1.5 pl-4 pr-1.5 text-sm font-medium text-zinc-900 transition-transform duration-500 ease-silk active:scale-[0.97] md:flex group">
+          <span>Pair your Mac</span>
+          <span class="flex h-7 w-7 items-center justify-center rounded-full bg-zinc-900/10 transition-transform duration-500 ease-silk group-hover:translate-x-0.5">
+            <svg viewBox="0 0 24 24" fill="none" stroke="currentColor" stroke-width="1.5" class="h-3.5 w-3.5 text-zinc-900"><path d="M5 12h14M13 6l6 6-6 6" stroke-linecap="round" stroke-linejoin="round"/></svg>
+          </span>
+        </a>
+
+        <!-- Hamburger -->
+        <button id="menuBtn" aria-label="Open menu" class="relative flex h-9 w-9 shrink-0 items-center justify-center rounded-full bg-white/[0.06] ring-1 ring-white/10 md:hidden">
```

HTMLHint (1.9.2)

[warning] The type attribute must be present on <button> elements.

(button-type-require)